src/api/app/controllers/staging/workflows_controller.rb

```diff
@@ -25,6 +25,7 @@ def destroy
     authorize @staging_workflow
 
     @staging_workflow.destroy!
+    render_ok
```

Comment:
I think we should also render something if we the staging could not be destroyed.